src/main/java/com/github/redouane59/twitter/TwitterClient.java

```diff
@@ -399,6 +404,43 @@ public boolean hideReply(final String tweetId, final boolean hide) {
         return result;
     }
 
+    @Override
+    public void startFilteredStream(Consumer<ITweet> consumer){
+        String      url    = this.urlHelper.getFilteredStreamUrl();
+        this.requestHelperV2.executeGetRequestWithConsumer(url, consumer);
+    }
+
+    @Override
+    public List<StreamRule> retrieveFilteredStreamRules() {
+        String       url    = this.urlHelper.getFilteredStreamRulesUrl();
+        StreamRulesDTO result = this.requestHelperV2.executeGetRequest(url, StreamRulesDTO.class).orElseThrow(NoSuchElementException::new);
+        return result.getData();
+    }
+
+    @Override
+    public StreamRule addFilteredStreamRule(String value, String tag){
+        String       url    = this.urlHelper.getFilteredStreamRulesUrl();
+        try {
+            StreamRule rule = StreamRule.builder().value(value).tag(tag).build();
+            String body = "{\"add\": ["+TwitterClient.OBJECT_MAPPER.writeValueAsString(rule)+"]}";
+            StreamRulesDTO result = this.requestHelperV2.executePostRequest(url, body, StreamRulesDTO.class).orElseThrow(NoSuchElementException::new);
+            if(result.getData()==null || result.getData().size()==0) throw new IllegalArgumentException();
+            return result.getData().get(0);
+        }
+        catch (JsonProcessingException e) {
+            LOGGER.error(e.getMessage(), e);
+            throw new IllegalArgumentException();
+        }
+    }
+
+    @Override
+    public StreamMeta deleteFilteredStreamRule(String ruleValue){
+        String       url    = this.urlHelper.getFilteredStreamRulesUrl();
+        String body = "{\"delete\": {\"values\": [\""+ruleValue+"\"]}}";
+        StreamRulesDTO result = this.requestHelperV2.executePostRequest(url, body, StreamRulesDTO.class).orElseThrow(NoSuchElementException::new);
+        return result.getMeta();
+    }
```

Comment:
I just installed `Saved Actions` plugin to avoid it, it should be enough.
thanks 
![image](https://user-images.githubusercontent.com/4500524/91550123-57d8a180-e928-11ea-810a-61c0540f2f48.png)